doc/source/whatsnew/v0.18.0.txt

```diff
@@ -112,3 +112,6 @@ Bug Fixes
 - Bug in ``.loc`` against ``CategoricalIndex`` may result in normal ``Index`` (:issue:`11586`)
 - Bug groupby on tz-aware data where selection not returning ``Timestamp`` (:issue:`11616`)
 - Bug in timezone info lost when broadcasting scalar datetime to ``DataFrame`` (:issue:`11682`)
+
+- Bugs in ``.drop`` and ``.groupby`` for dataframe with multi-indexed columns (:issue:`11640`)
+- Bug in ``.groupby``: a ``KeyError`` was not raised for a wrong column if there was only one row in the dataframe (:issue:`11640`)
```

Comment:
It's not a bug in `.groupby(...).drop(...)`, it really is a bug showing up separately in both `.groupby` and `.drop`. I'll make the What's New clearer.